tests/test_container.rs

```diff
@@ -0,0 +1,58 @@
+use restate_test_utils::test_utils::TestContainer;
```

Comment:
Could you move this test in `restate-test-utils`? This way you don't need a dependency on `restate-test-utils` in the root Cargo.toml. I'm afraid this might cause some issues